<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editUSER</class>
 <widget class="QDialog" name="editUSER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Người dùng</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/users.png</normaloff>:/images/users.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>504</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Công việc</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CMND/MSSV</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>24</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>32</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Họ tên</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nghề nghiệp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>46</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tài khoản</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ngày sinh</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_taikhoan">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_hoten">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cmnd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ngaysinh">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_congviec">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nghenghiep">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>411</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <zorder>lineEdit_cmnd</zorder>
      <zorder>lineEdit_email</zorder>
      <zorder>lineEdit_ngaysinh</zorder>
      <zorder>lineEdit_congviec</zorder>
      <zorder>lineEdit_nghenghiep</zorder>
      <zorder>lineEdit_taikhoan</zorder>
      <zorder>lineEdit_hoten</zorder>
      <zorder>label_6</zorder>
      <zorder>label_3</zorder>
      <zorder>label_4</zorder>
      <zorder>label_2</zorder>
      <zorder>label_7</zorder>
      <zorder>label</zorder>
      <zorder>label_5</zorder>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_update">
        <property name="text">
         <string>Cập nhật</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Xoá người dùng</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_del">
        <property name="text">
         <string>Xoá tài khoản</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_lock">
        <property name="text">
         <string>Khoá tài khoản</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="text">
         <string>Reset mật khẩu</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;THÔNG TIN NGƯỜI DÙNG&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
